model/src/main/kotlin/ScanSummary.kt

```diff
@@ -94,17 +92,15 @@ class ScanSummaryDeserializer : StdDeserializer<ScanSummary>(OrtIssue::class.jav
             null
         }
 
-    private inline fun <reified T : Any> JsonNode.readValues(property: String, kClass: KClass<T>): List<T> =
-        this[property]?.map {
-            jsonMapper.treeToValue(it, kClass.java)
-        }.orEmpty()
+    private inline fun <reified T : Any> JsonNode.readValues(property: String): List<T> =
+        this[property]?.mapNotNull { jsonMapper.treeToValue<T>(it) }.orEmpty()
```

Comment:
Why did you change this to `mapNotNull`, can there be null values?

Reply:
Yes, the function signature is

    inline fun <reified T> ObjectMapper.treeToValue(n: TreeNode): T? = treeToValue(n, T::class.java)

i.e. it has a nullable return type.

Reply:
Thanks.